ui/src/views/Hooks/ListHooks/index.jsx

```diff
@@ -91,12 +85,12 @@ export default class ListHooks extends Component {
             key={hookSearch}
             defaultExpanded={Boolean(hookSearch)}
             listItemProps={{ color: classes.listItemProps }}
-            searchTerm={hookSearch}
+            searchTerm={hookSearch || null}
             softSearch
             tree={tree}
             onEmptySearch={
-              <Typography variant="h3" color="secondary">
-                No hooks are defined {hookSearch}
+              <Typography variant="subtitle1">
+                 {`No hooks are defined ${hookSearch}`}
               </Typography>
```

Comment:
I had added a string logic here but I am still confused about the no display hooks logics.